.github/workflows/commons-ci.yml

```diff
@@ -36,19 +39,23 @@ jobs:
         with:
           fetch-depth: 2
 
-      - name: License check(RAT)
+      - name: Use staged maven repo settings
+        if: ${{ env.USE_STAGE == 'true' }}
```

Comment:
we need visit stage here?

Reply:
commons-rpc depends on commons-commons, which currently only exists in the maven stage repository

Reply:
TODO: reset use stage to false later